nemo/collections/asr/models/asr_model.py

```diff
@@ -154,18 +154,53 @@ def input_module(self):
     def output_module(self):
         return self.decoder
 
-    def forward_for_export(self, input, length=None):
+    def forward_for_export(self, input, length=None, cache_last_channel=None, cache_last_time=None):
+        """
+        This forward is used when we need to export the model to ONNX format.
+        Inputs cache_last_channel and cache_last_time are needed to be passed for exporting streaming models.
+        When they are passed, it just passes the inputs through the encoder part and currently the ONNX conversion does not fully work for this case.
+        Args:
+            input: Tensor that represents a batch of raw audio signals,
+                of shape [B, T]. T here represents timesteps.
+            length: Vector of length B, that contains the individual lengths of the audio sequences.
+            cache_last_channel: Tensor of shape [N, B, T, H] which contains the cache for last channel layers
+            cache_last_time: Tensor of shape [N, B, H, T] which contains the cache for last time layers
+                N is the number of such layers which need caching, B is batch size, H is the hidden size of activations,
+                and T is the length of the cache
+
+        Returns:
+            the output of the model
+        """
         if hasattr(self.input_module, 'forward_for_export'):
-            encoder_output = self.input_module.forward_for_export(input, length)
+            if cache_last_channel is None and cache_last_time is None:
```

Comment:
Note that for now export test has been disabled, @borisfom  we need to check that these modifications are exportable by re-enabling the test

Reply:
Going to test the export later. I need to verify it with the latest changes locally first.

Reply:
Ok leaving this comment unresolved for later check then.